src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -126,6 +126,42 @@ public ActionPlanDTO createActionPlan(
     return responseEntity.getBody();
   }
 
+  @Retryable(
+      value = {RestClientException.class},
+      maxAttemptsExpression = "#{${retries.maxAttempts}}",
+      backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
+  @Override
+  public List<ActionPlanDTO> getActionPlansBySelectorsSocial(final String collectionExerciseId) {
+    log.debug(
+        "Retrieving action plan for selectors, " + "collectionExerciseId: {}",
```

Comment:
why break the string into two separate strings?

Reply:
:+1: Fixed, I think this was originally done by the auto-formatting